ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/remove_workload.yml

```diff
@@ -13,5 +13,5 @@
 # --------------------------------------------
 - name: remove_workload tasks complete
   debug:
-    msg: "Remove Workload tasks completed successfully."
-  when: not silent|bool
+   msg: "Remove Workload tasks completed successfully."
```

Comment:
Removed space?

Reply:
yamllint wants indent 3 at that spot. If I leave the space there, it gives me an error.